terragrunt/aws/buckets/great_expectations.tf

```
@@ -0,0 +1,18 @@
+module "gx_bucket" {
+  source            = "github.com/cds-snc/terraform-modules//S3?ref=v10.4.6"
+  bucket_name       = "cds-data-lake-gx-${var.env}"
+  billing_tag_value = var.billing_tag_value
+
+  logging = {
+    target_bucket = module.log_bucket.s3_bucket_id
+    target_prefix = "gx/"
+  }
+
+  lifecycle_rule = [
+    local.lifecycle_remove_noncurrent_versions
```

While looking at this I realized it would be good idea to also transfer objects to a cheaper storage class after a delay to save some money:
```suggestion
    local.lifecycle_remove_noncurrent_versions,
    local.lifecycle_transition_storage
```